docs/product/components/pagination.html

```diff
@@ -29,72 +29,113 @@
 </section>
 <section class="stacks-section">
     {% header "h2", "Example" %}
+
+    {% tip, "warning", "mb24" %}
```

Comment:
I don't think this note makes a lot of sense here in isolation since it is unlikely people will remember the old markup. 
Maybe we could have a section called legacy example where we add the deprecation warning and that it will be removed in future releases.

This is what I am suggesting from a content hierarchy perspective:

- Example 
  - New markup example
 
- Legacy Example
  - Deprecation warning message
  - Old markup example 
  
  What do you think?

Reply:
Sounds good to me. I've made the change